xtevent/xtevent.sthlp

```diff
@@ -192,9 +192,12 @@ trend (_ttrend) to the regression. {opt (gmm)} uses the GMM to compute the trend
 {opt saveov:erlay} saves estimations for the overlay plot produced by {opt xteventplot, overlay(trend)}.
 
 {phang}
-{opt savek(stub)} saves variables for event-time dummies, event-time and trends. Event-time dummies are stored as {it: stub}_eq_m# for the dummy
+{opt savek(stub, [subopt])} saves variables for event-time dummies, event-time and trends. Event-time dummies are stored as {it: stub}_eq_m# for the dummy
 variable # periods before the policy change, and {it:stub}_p# for the dummy variable # periods after the policy change. The dummy variable for
-the policy change time is {it:stub}_p0. Event time is stored as {it:stub}_evtime. The trend is stored as {it:stub}_trend.
+the policy change time is {it:stub}_p0. Event time is stored as {it:stub}_evtime. The trend is stored as {it:stub}_trend. The following ca be passed as a suboption:
+
+{phang2}
+{opt noe:stimate} saves variables for event-time dummies, event-time and trends without estimating the model. This can be useful if the users want to customize their regressions and plots.

```

Comment:
@Constantino-Carreto-Romero  See https://github.com/JMSLab/xtevent/pull/94/files#r948491879

Also, in 200 "This is useful if the user ..."